maintainer_fixtures.json

```diff
@@ -0,0 +1,5252 @@
+[
+    {
+        "model": "ports.Maintainer",
+        "pk": 0,
+        "fields": {
+            "github": "ryandesign",
```

Comment:
This file should go. Maintainers should be read and dynamically populated / updated from portindex json data.

Reply:
This file is dynamically generated from the `portindex` using the scripts inside `parsing-scripts`. But again here also, I kept one copy to have some initial data.

There will be two types of scripts: [1] those which generate initial data in the form of Django fixtures, it is very efficient way of populating database with initial data. And given the large amount of data in portindex, it would be good to populate the database with the initial data.

[2] After the initial data is in the database, a script that would take the difference of portindex will keep our database updated. I had such script earlier, but since now the approach is changed and we probably will use `portindex -c` or `portindex -diff` for this purpose, the new script is not ready yet.